tests/v2/validation/provisioning/README.md

```diff
@@ -21,5 +21,5 @@ rancher:
 From there, your config should contain the tests you want to run (provisioningInput), tokens and configuration for the provider(s) you will use, and any additional tests that you may want to run. Please use one of the following links to continue adding to your config for provisioning tests:
 
 1. [RKE1 Provisioning](rke1/README.md)
-2. [RKE2 Provisioning](rke2/README.md)
+2. [RKE2/K3s Provisioning](rke2/README.md)
```

Comment:
Why is this together? `k3s` has its own package so we should just add a new line that references that provisioning, right?